website/site/content/platforms/arduinouno.md

```diff
@@ -0,0 +1,210 @@
+			<td>True if the button is pressed in this very right moment. False otherwise.</td>
+		</tr>
+    </tbody>
+</table>
+
+
+#### Events
+<table>
+    <thead>
+        <tr>
+            <td>Name</td>
+            <td>Description</td>
+        </tr>
+    </thead>
+    <tbody>
+		<tr>
+			<td><div class="highlight"><pre><b>pressed</b></pre></div></td>
+			<td>Fires after the button was pressed.</td>
+		</tr>
+		<tr>
+			<td><div class="highlight"><pre><b>released</b></pre></div></td>
+			<td>Fires after the button was released.</td>
+		</tr>
+    </tbody>
+</table>
+
+## Sensor: button_two (Button)
+
+
+#### Modalities
+<table>
+    <thead>
+        <tr>
+            <td>Name</td>
+            <td>Description</td>
+        </tr>
+    </thead>
+    <tbody>
+		<tr>
+			<td><div class="highlight"><pre><b>is_pressed</b> : <span class="kt">bool</span></pre></div></td>
+			<td>True if the button is pressed in this very right moment. False otherwise.</td>
+		</tr>
+    </tbody>
+</table>
+
+
+#### Events
+<table>
+    <thead>
+        <tr>
+            <td>Name</td>
+            <td>Description</td>
+        </tr>
+    </thead>
+    <tbody>
+		<tr>
+			<td><div class="highlight"><pre><b>pressed</b></pre></div></td>
+			<td>Fires after the button was pressed.</td>
+		</tr>
+		<tr>
+			<td><div class="highlight"><pre><b>released</b></pre></div></td>
+			<td>Fires after the button was released.</td>
+		</tr>
+    </tbody>
+</table>
+
+## Buses: GPIO
+Can be used to write and read digital ports, as known from the Arduino Uno
+
+### Signals
+<table>
+    <thead>
+        <tr>
+            <td>Name</td>
+            <td>Description</td>
+            <td>Parameters</td>
+        </tr>
+    </thead>
+    <tbody>
+		<tr>
+			<td><div class="highlight"><pre><b>pinMode</b> : <span class="kt">bool</span></pre></div></td>
+			<td></td>
+			<td>
+				<ul>
+				<li>
+					<div class="highlight"><pre> <b>pin</b> : <span class="kt">LedColor</span></pre></div>
+				</li>
+				</ul>
+			</td>
+		</tr>
+    </tbody>
+</table>
+
+##Example 
+
+This example uses the first button connected to the hardware Pin 1 and two leds connected to pin 12 and 13. 
+The yellow led can get turned off by pressing the button one and turned off again by pressing the button again. 
+The red led will get turned on for one second and then turned off for one second.
+
+```TypeScript
+package main;
+import platforms.arduino.uno;
+
+setup hmi : GPIO {
+	var yellow = pinMode(p13, OUTPUT);
+	var red = pinMode(p12, OUTPUT);
+}
+
+every button_one.pressed {
+	if(hmi.yellow.read() == false) {
+		hmi.yellow.write(true);
+	} else {
+		hmi.yellow.write(false);
+	}
+}
+
+every 1 second {
+	if(hmi.red.read() == false) {
+		hmi.red.write(true);
+	} else {
+		hmi.red.write(false);
+	}
+}
+```
+
+##Interrupt controlled event loop
+
+Instead of using an event queue, as the XDK 110 does, a control with boolean flags is implemented. 
```

Comment:
Please use active voice in texts ("we implemented control flow using boolean flags", rather than "control flow with boolean flags is implemented".
See Strunk and White's _Elements of Style_.